garrysmod/lua/includes/modules/widget.lua

```diff
@@ -145,6 +141,24 @@ hook.Add( "PostDrawEffects", "RenderWidgets", function()
 
 end )
 
+hook.Add( "OnEntityCreated", "CreateWidgets", function( ent )
+	timer.Simple( 1, function( ) -- why is this needed? no one may ever know!

```

Comment:
How about not using any timer at all?

Anyway, timer with 0 may not actually help because it can be called at this frame, not at the next frame. This has been discussed somewhere as far as I can remember. The solution was just using a very little delay (like 0.001) for the timer instead of 0.